boards/arm/nrf52/nrf52832-mdk/configs/nsh/defconfig

```diff
@@ -0,0 +1,45 @@
+#
+# This file is autogenerated: PLEASE DO NOT EDIT IT.
+#
+# You can use "make menuconfig" to make any modifications to the installed .config file.
+# You can then do "make savedefconfig" to generate a new defconfig file that includes your
+# modifications.
+#
+# CONFIG_ARCH_FPU is not set
+# CONFIG_NSH_DISABLE_IFCONFIG is not set
+# CONFIG_NSH_DISABLE_PS is not set
+CONFIG_ARCH="arm"
+CONFIG_ARCH_BOARD="nrf52832-sparkfun"
+CONFIG_ARCH_BOARD_NRF52832_SPARKFUN=y
```

Comment:
```suggestion
CONFIG_ARCH_BOARD="nrf52832-mdk"
CONFIG_ARCH_BOARD_NRF52832_MDK=y
```

Reply:
Good catch. Don't know why it ended up like this since I used the refresh tool.

Reply:
I added this change to last commit.